{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2016-9462",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "nextcloud",
            "product": {
              "product_data": [
                {
                  "product_name": "nextcloud",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "9.0.51"
                      }
                    ]
                  }
                }
              ]
            }
          },
          {
            "vendor_name": "owncloud",
            "product": {
              "product_data": [
                {
                  "product_name": "owncloud",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "8.0.13"
                      },
                      {
                        "version_value": "8.1.8"
                      },
                      {
                        "version_value": "8.2.6"
                      },
                      {
                        "version_value": "9.0.3"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-284"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www.securityfocus.com/bid/97285"
        },
        {
          "url": "https://github.com/nextcloud/server/commit/1208953ba1d4d55a18a639846bbcdd66a2d5bc5e"
        },
        {
          "url": "https://github.com/owncloud/core/commit/23383080731d092e079986464a8c4c9ffcb79f4c"
        },
        {
          "url": "https://github.com/owncloud/core/commit/3b056fa68ce502ceb0db9b446dab3b9e7b10dd13"
        },
        {
          "url": "https://github.com/owncloud/core/commit/c93eca49c32428ece03dd67042772d5fa62c8d6e"
        },
        {
          "url": "https://github.com/owncloud/core/commit/d31720b6f1e8c8dfeb5e8805ab35ad7c8000b2f1"
        },
        {
          "url": "https://hackerone.com/reports/146067"
        },
        {
          "url": "https://nextcloud.com/security/advisory/?id=nc-sa-2016-005"
        },
        {
          "url": "https://owncloud.org/security/advisory/?id=oc-sa-2016-015"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Nextcloud Server before 9.0.52 & ownCloud Server before 9.0.4 are not properly verifying restore privileges when restoring a file. The restore capability of Nextcloud/ownCloud was not verifying whether a user has only read-only access to a share. Thus a user with read-only access was able to restore old versions."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:nextcloud:nextcloud",
            "cpe23Uri": "cpe:2.3:a:nextcloud:nextcloud:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "9.0.51"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:owncloud:owncloud",
            "cpe23Uri": "cpe:2.3:a:owncloud:owncloud:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "8.0.13"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:owncloud:owncloud",
            "cpe23Uri": "cpe:2.3:a:owncloud:owncloud:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "8.1.8"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:owncloud:owncloud",
            "cpe23Uri": "cpe:2.3:a:owncloud:owncloud:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "8.2.6"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:owncloud:owncloud",
            "cpe23Uri": "cpe:2.3:a:owncloud:owncloud:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "9.0.3"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "version": "3.0",
        "vectorString": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:N",
        "attackVector": "NETWORK",
        "attackComplexity": "LOW",
        "privilegesRequired": "LOW",
        "userInteraction": "NONE",
        "scope": "UNCHANGED",
        "confidentialityImpact": "NONE",
        "integrityImpact": "LOW",
        "availabilityImpact": "NONE",
        "baseScore": 4.3,
        "baseSeverity": "MEDIUM"
      },
      "exploitabilityScore": 2.8,
      "impactScore": 1.4
    },
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:N/AC:L/Au:S/C:N/I:P/A:N)",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "SINGLE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "PARTIAL",
        "availabilityImpact": "NONE",
        "baseScore": 4.0
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8.0,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2017-03-28T02:59Z",
  "lastModifiedDate": "2017-04-04T01:59Z"
}